src/transformers/modeling_tf_utils.py

```diff
@@ -809,25 +809,30 @@ def resize_token_embeddings(self, new_num_tokens=None) -> tf.Variable:
 
         return model_embeds
 
-    def _get_word_embedding_weight(self, embedding_layer):
-        if hasattr(embedding_layer, "word_embeddings"):
-            return embedding_layer.word_embeddings
-        elif hasattr(embedding_layer, "weight"):
-            return embedding_layer.weight
-        elif hasattr(embedding_layer, "decoder"):
-            return embedding_layer.decoder
-        else:
-            # Here we build the word embeddings weights if not exists.
-            # And then we retry to get the attribute once built.
-            self(self.dummy_inputs)
-            if hasattr(embedding_layer, "word_embeddings"):
-                return embedding_layer.word_embeddings
-            elif hasattr(embedding_layer, "weight"):
-                return embedding_layer.weight
-            elif hasattr(embedding_layer, "decoder"):
-                return embedding_layer.decoder
-            else:
-                return None
+    def _get_word_embedding_weight(model, embedding_layer):
+        embeds = getattr(embedding_layer, "weight", None)
+
+        if embeds is not None:
+            return embeds
```

Comment:
```suggestion
        embeds = getattr(embedding_layer, "weight", None)
        if embeds is not None:
            return embeds
```
I'd group those together (and same for the others below) as this is a lot of new lines :-)